pkgs/development/libraries/ngtcp2/default.nix

```
@@ -48,11 +48,12 @@ stdenv.mkDerivation rec {
     inherit curlHTTP3;
   };
 
-  meta = with lib; {
+  meta = {
     homepage = "https://github.com/ngtcp2/ngtcp2";
-    description = "ngtcp2 project is an effort to implement QUIC protocol which is now being discussed in IETF QUICWG for its standardization";
-    license = licenses.mit;
-    platforms = platforms.unix;
-    maintainers = with maintainers; [ izorkin ];
+    changelog = "https://github.com/ngtcp2/ngtcp2/releases/tag/${finalAttrs.src.tag}";
```

```suggestion
    changelog = "https://github.com/ngtcp2/ngtcp2/releases/tag/v${finalAttrs.version}";
```

`finalAttrs.src.tag` shouldn't be used here since this would break if someone overrides the `src`

---

r-ryantm will change this back, so I'd raise the issue there.

---

I think this only applies to python packages (see https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/up/update-python-libraries/update-python-libraries.py#L506)